### ISSUE TITLE
[v1.2-Elements-Input.Text.InlineAction.json]: Reply controls associated with 'Text input with an inline action' and 'Text input with an inline action with no icon' text field is not accessible by voice over using swipe gesture.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#7025]
https://microsoft.visualstudio.com/OS/_workitems/edit/37853130

**Comment by vagpt** [comment]
Closing this issue as elements under 1.2 is not available.
